subsys/net/lib/openthread/platform/radio.c

```diff
@@ -956,12 +956,17 @@ otError otPlatRadioSetTransmitPower(otInstance *aInstance, int8_t aPower)
 	return OT_ERROR_NONE;
 }
 
+uint64_t otPlatTimeGet(void)
+{
+	return radio_api->get_time(radio_dev);
```

Comment:
Many of the `radio_api` implementations don't have `get_time` set, shouldn't this be checked or even fallback to `k_uptime_get()` for example?

Reply:
Good point, thanks.